### PR TITLE
p5-mail-spamassassin: Update SpamAssassin to v3.4.4

### PR DESCRIPTION
**Pull request by grumpybozo**
#### Description

Routine version update. Resolves 4  CVEs:

- CVE-2019-12420 for Multipart Denial of Service Vulnerability
- CVE-2018-11805 for nefarious CF files can be configured to run system commands without any output or errors.
- CVE-2020-1931 for Nefarious rule configuration (.cf) files can be configured to run system commands with warnings.
- CVE-2020-1930 for Nefarious rule configuration (.cf) files can be configured to run system commands with sa-compile.

Also resolves a very large number of bugs and adds functionality. Details are in the package and the announcements of 3.4.3 and 3.4.4:

https://mail-archives.apache.org/mod_mbox/spamassassin-announce/201809.mbox/%3cc44ca0f1-cba9-b129-20b2-ba59816cfd13@apache.org%3e
https://mail-archives.apache.org/mod_mbox/spamassassin-announce/202001.mbox/%3cebccac74-32c9-4c3c-e14d-a7b4f40d74cc@apache.org%3e

Trac: https://trac.macports.org/ticket/60052

###### Type(s)

- [x] bugfix
- [x] enhancement
- [x] security fix

###### Tested on

macOS 10.14.6 18G3020
Xcode 11.0 11A420a 

macOS 10.6.8 10K549
Xcode 3.2.6 DevToolsSupport-1806.0 10M2518 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`? (Tests not enabled in Portfile, done in release process)
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: perl/p5-mail-spamassassin/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26 5.28 5.30
-perl5.setup         Mail-SpamAssassin 3.4.2
+perl5.setup         Mail-SpamAssassin 3.4.4
 revision            1
```
Comment (mf2k) [inline perl/p5-mail-spamassassin/Portfile]: The `revision` line needs to be reset to 0 when upgrading the `version`.

**Comment by mf2k** [comment]
The commit message needs to updated to [follow the guidelines](https://trac.macports.org/wiki/CommitMessages).

File: perl/p5-mail-spamassassin/Portfile (hunk 1 above)
Comment (grumpybozo) [reply perl/p5-mail-spamassassin/Portfile]: Removed revision line (this seems to be the more common approach.)

Comment (mf2k) [reply perl/p5-mail-spamassassin/Portfile]: It is now preferred to leave it in with 0.

**Comment by mf2k** [comment]
The commit messages need to be squashed into 1 and still do not conform to the guidelines.

**Comment by grumpybozo** [comment]
Resubmitting with squashed commits

**Pull request closed without merging**